shared/tracker2/bio/container.tsx

```diff
@@ -1,51 +1,48 @@
 import * as Container from '../../util/container'
 import * as Constants from '../../constants/tracker2'
 import * as RouteTreeGen from '../../actions/route-tree-gen'
-import Bio, {Props} from '.'
+import Bio from '.'
 
 type OwnProps = {
   inTracker: boolean
   username: string
 }
 
-const mapStateToProps = (state: Container.TypedState, ownProps: OwnProps) => {
-  const d = Constants.getDetails(state, ownProps.username)
-  const common = {
-    blocked: d.blocked,
-    hidFromFollowers: d.hidFromFollowers,
-    username: ownProps.username,
-  }
-  if (d.state === 'notAUserYet') {
-    const nonUser = Constants.getNonUserDetails(state, ownProps.username)
-    return {
-      ...common,
-      bio: nonUser.bio,
-      followThem: false,
-      followsYou: false,
-      fullname: nonUser.fullName,
-      sbsDescription: nonUser.description,
+export default Container.namedConnect(
+  (state: Container.TypedState, ownProps: OwnProps) => {
+    const d = Constants.getDetails(state, ownProps.username)
+    const common = {
+      blocked: d.blocked,
+      hidFromFollowers: d.hidFromFollowers,
+      username: ownProps.username,
     }
-  } else {
-    return {
-      ...common,
-      bio: d.bio,
-      followThem: Constants.followThem(state, ownProps.username),
-      followersCount: d.followersCount,
-      followingCount: d.followingCount,
-      followsYou: Constants.followsYou(state, ownProps.username),
-      fullname: d.fullname,
-      location: d.location,
+    if (d.state === 'notAUserYet') {
```

Comment:
Is this a special value that is used when `getDetails` doesn't find anything? Can we make it a constant in `Constants`?

Reply:
this is typed, its `DetailsState` (and unchanged). TS will complain if its not part of the enum